### ISSUE TITLE
I want to change the title (meta data). 

### ISSUE THREAD

**Issue by iToshk** [weshatheleopard/rubyXL#395]
Looking for a way to change excel title property for displaying filename on the browsers.

**Comment by weshatheleopard** [comment]
Sorry, this is the place to report bugs, not free educational seminar. Try Stack Overflow.

**Comment by iToshk** [comment]
oh, Sorry.
I found the solution.
I looked for the methods in a workbook instance and I found that the root method has core_properties and inside there, dc_title was the one I looked for.

In short, I could get the title value by `wb.root.core_properties.dc_title.value`.
And set value by `wb&.root&.core_properties&.dc_title = RubyXL::StringNode.new(:value => title)`

I found that there are no methods for changing dc_title here.
https://github.com/weshatheleopard/rubyXL/blob/master/lib/rubyXL/objects/document_properties.rb

So, maybe I can create methods and specs for that. and creating commit and PR?

**Comment by weshatheleopard** [comment]
See, you were able to accomplish exactly what you needed.

Are you saying you want to create a _convenience method_ to do that? Certainly, go ahead.

**Comment by iToshk** [comment]
@weshatheleopard Yeah, actually the reason why I created this issue was that I wanted to ask you about making PR and commit about **a convenience method**. Thank you for the quick reply! (I didn't expect it so I was in the middle of writing comments)
(Indeed, I had already solved my problem and just wanted to know if it's good for making PR with it.)

Then, I'll make PR for _a convenience method_ later. thank you!

Thank you for this awsome gem!
Best,
